File: normalandroid/src/main/java/com/example/normalandroid/MainActivity.java
package com.example.normalandroid;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.annotation.SuppressLint;
import android.app.Notification;
import android.app.NotificationChannel;
import android.app.NotificationManager;
import android.content.Intent;
import android.content.SharedPreferences;
import android.graphics.BitmapFactory;
import android.net.Uri;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.TextView;

import java.net.URI;

@SuppressLint("MissingInflatedId")
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView textView = findViewById(R.id.text1111);
        textView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Log.d("Ning" , "onClick");

                NotificationManager manager = (NotificationManager) getSystemService(NOTIFICATION_SERVICE);
                if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
                    NotificationChannel channel = new NotificationChannel("1" , "重要信息" , NotificationManager.IMPORTANCE_HIGH);
                    manager.createNotificationChannel(channel);

                    Notification notification = new Notification.Builder(MainActivity.this , "1")
                            .setContentTitle("This is content title")
                            .setContentText("This is content text")
                            .setWhen(System.currentTimeMillis())
                            .setSmallIcon(R.mipmap.ic_launcher)
                            .setLargeIcon(BitmapFactory.decodeResource(getResources(),R.mipmap.ic_launcher))
                            .build();
                    manager.notify(1 , notification);
                }
            }
        });
    }
}

File: layouttest/src/main/java/com/example/layouttest/MainActivity.kt
package com.example.layouttest

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.constraintlayout.widget.ConstraintLayout
import okhttp3.FormBody
import okhttp3.RequestBody

class MainActivity : AppCompatActivity() {
    val API_KEY = "5GHSlGSkqlDiGA57zWmilkwp"
    val SECRET_KEY = "UAbadAXq7Evl1qbg9j9aWGcXibZ8oNH8"

    var formBody : RequestBody = FormBody.Builder()
        .build()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


    }

    
}